plugins/pulp_rpm/plugins/migrations/0001_rpm_add_export_distributor.py

```diff
@@ -41,7 +42,7 @@ def _migrate_rpm_repositories():
             iso_distributor = collection.find_one(
                 {'repo_id': repo_distributor['repo_id'], 'distributor_type_id': 'iso_distributor'})
             if iso_distributor is not None:
-                collection.remove(iso_distributor)
+                collection.remove(iso_distributor, safe=True)
```

Comment:
@rbarlow, yes, @dkliban did put the necessary machinery in place, and I actually removed the instances of this. Must have had a merge conflict or something to put this one back in.